php-checks/src/main/java/org/sonar/php/checks/utils/CheckUtils.java

```diff
@@ -357,6 +361,19 @@ public static boolean hasAnnotation(Tree declaration, String annotation) {
     return false;
   }
 
+  public static List<String> getAttributeNames(Tree declaration) {
+    List<String> result = new ArrayList<>();
+    if (declaration instanceof HasAttributes) {
+      List<AttributeGroupTree> attributeGroupTrees = ((HasAttributes) declaration).attributeGroups();
+      for (AttributeGroupTree attributes : attributeGroupTrees) {
+        for (AttributeTree attributeTree : attributes.attributes()) {
+          result.add(attributeTree.name().unqualifiedName());
```

Comment:
Why the `unqualifiedName()` instead of the `fullName()` Does it make a difference in this case?